src/Svg/Tag/AbstractTag.php

```diff
@@ -166,7 +166,13 @@ protected function applyTransform($attributes)
                         break;
 
                     case "rotate":
-                        $surface->rotate($t[1]);
+                        if (isset($t[2]) || isset($t[3])) {
```

Comment:
Could you check only if the third item is defined? It'll throw a warning if only the second one is defined.

Reply:
Thanks for your hint. I have updated the PR know. The condition does only check for existance of the second item now, and set `0` as default value for the third item, so that `rotate(45, 2)` can be processed, too.

What do you think?